queue/queue.py

```diff
@@ -3,13 +3,14 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
```

Comment:
I would strongly consider using a LinkedList() class instead of a list/array for the storage structure for the Queue.

That is, the LinkedList() will allow us to have constant time operations for adding/removing where an array/list will not.